vm_insnhelper.c

```diff
@@ -2701,16 +2701,75 @@ ci_missing_reason(const struct rb_callinfo *ci)
     return stat;
 }
 
```

Comment:
Small comment: In commit message, `method missing` without any note can be misleading because it does not mean general method missing, but the given Symbol does not has corresponding ID.

The algorithm seems ok, but the name `vm_send_this_symbol()` is not fit to me because the "this" is not general terminology in MRI code, IMO.

Reply:
OK, that sounds a valid concern.  What do you suggest?

Reply:
For comment, adding short note seems fine.

For function name, simply `vm_call_symbol()` seems enough.